.storybook/main.js

```diff
@@ -1,59 +1,22 @@
-/* eslint-disable import/no-extraneous-dependencies */
-const vueJsx = require('@vitejs/plugin-vue-jsx');
-const { mergeConfig } = require('vite');
-
 module.exports = {
-  framework: '@storybook/vue3',
+  framework: {
+    name: '@storybook/vue3-vite',
+    options: {},
+  },
   core: {
-    builder: '@storybook/builder-vite',
     disableTelemetry: true,
     enableCrashReports: false,
   },
-  staticDirs: [
-    '../public',
-    '../src/assets',
-    {
-      from: '../node_modules/svg-country-flags/png100px/',
-      to: '/icons/country-flags/100',
-    },
-  ],
-  stories: [
-    '../src/**/*.story.mdx',
-    '../src/**/*.story.@(js|jsx|ts|tsx)',
-  ],
-  addons: [
-    '@storybook/addon-a11y',
-    '@storybook/addon-cssresources',
-    '@storybook/addon-essentials',
-    '@storybook/addon-links',
-    {
-      name: '@storybook/addon-postcss',
-      options: {
-        postcssLoaderOptions: {
-          // eslint-disable-next-line global-require
-          implementation: require('postcss'),
-        },
-      },
-    },
-  ],
-  viteFinal: (config) => {
-    const reactPluginNameRegex = /^vite:react-/;
-    const reactPluginsArrayIndex = config.plugins.findIndex(plugin => Array.isArray(plugin) && plugin.every(p => p.name.match(reactPluginNameRegex)));
-
-    if (reactPluginsArrayIndex === -1) {
-      throw new Error('Unable to patch plugins - cannot find vite:react plugins. Is this fix still necessary?');
-    }
-
-    // remove react plugins and replace them with vue plugins
-    // see https://github.com/storybookjs/storybook/issues/21681
-    // remove after migrating to SB 7
-    const pluginsWithoutReact = config.plugins.filter((_, index) => index !== reactPluginsArrayIndex);
-    return mergeConfig({
-      ...config,
-      plugins: [
-        ...pluginsWithoutReact,
-        vueJsx(),
-      ],
-    });
+  staticDirs: ['../public', '../src/assets', {
+    from: '../node_modules/svg-country-flags/png100px/',
+    to: '/icons/country-flags/100',
+  }],
+  stories: ['../src/**/*.story.mdx', '../src/**/*.story.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-a11y', '@storybook/addon-cssresources', '@storybook/addon-essentials', '@storybook/addon-links', '@storybook/addon-styling'],
+  docs: {
+    // Docs tab was changed for a docs page in the sidebar, to
+    // create this new page we need the following property enabled
+    // (this apply to all stories)
+    autodocs: true,
```

Comment:
Docs are still there. Have you tried to run `npm run test:visual:all`?

![image](https://github.com/Ebury/chameleon/assets/5939240/ba037f03-cfba-4d2c-aa7b-53b473133802)

Reply:
That's how is supposed to work:

![image](https://github.com/Ebury/chameleon/assets/77344353/dc9b2cfc-abb0-4b75-adca-38a131180d4b)

More info about the new docs position [here](https://storybook.js.org/blog/storybook-7-docs/)

Reply:
And regarding the visual regression tests, it fails this branch but also in `master`, is this supposed to happen?

Reply:
What is the error during the failing?

Reply:
This is the screenshot:

![image](https://github.com/Ebury/chameleon/assets/77344353/42d6a5ca-04a8-41e0-860d-b67ce45732bd)

This is the message from the JUnit report:

`Image was 0.00024112654320987653% different from saved snapshot with 3 different pixels.
See diff for details: /home/raulaguilera/Documents/chameleon/tests/integration/snapshots/visual-regression.spec.js/__diff_output__/button--props-dark__chrome__knob-0__anchors.diff.png`

And this is the diff output:

![image](https://github.com/Ebury/chameleon/assets/77344353/212fccad-d519-46f4-8238-41380af0ffdc)

Reply:
Error you have is a flaky screenshot error. visual-regression.spec.js uses story names from sidebar to generate screenshots. We had no ones before for `Docs`, now we have. What we are going to do with those screenshots for `Docs` stories?

Reply:
Is there any way to regenerate the snapshots? I want to test a couple of things because I believe that the new Docs pages does not affect our current test since they are not stories as you can see [here](https://storybook.js.org/docs/vue/writing-stories/naming-components-and-hierarchy). (There are also a problem with the way we get stories using `getStoriesJsonData` due to API changes)